docs/src/userguide/faq.rst

```diff
@@ -0,0 +1,1018 @@
+
+The distribution size, and the size per module, will probably increase when splitting a module because there are some things that Cython has to copy into each module. There is a [[feature request|https://github.com/cython/cython/issues/2356]] that would mitigate this.
+
+C calls between modules are slightly slower than C calls inside of a module, simply because the C compiler cannot optimise and/or inline them. You will have to use shared .pxd declarations for them, which will then call through a function pointer. If modules use a functional split, however, this should not hurt too much. It might still be a good idea to create a shared .pxd file (or .pxi) with inline functions for performance critical code that is used in multiple modules.
+
+When splitting an existing module, you will also have to deal with the API changes. Leaving some legacy imports here and there, or turning a module into a package that merges the module namespaces back together via imports, might prevent code breakage for users of your original module when you move names around and redistribute them across multiple modules.
+
+----------
+
+What is the difference between ``PyObject*`` and ``object``?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Answer**: A variable of type ``PyObject*`` is a simple C pointer, just like ``void*``. It is not reference counted, which is sometimes referred to as a borrowed reference. An ``object`` variable is an owned reference to a Python object. You can convert one into the other by casting:
+
+::
+
+    from cpython.ref cimport PyObject
+
+    py_object = [1,2,3]
+
+    cdef PyObject* ptr = <PyObject*>py_object
+
+    cdef object l = <object>ptr    # this increases the reference count to the list
+
+Note that the lifetime of the object is only bound to its owned references, not to any C pointers that happen to point to it. This means that ``ptr`` in the example above becomes invalid as soon as the last reference to the object dies:
+
+::
+
+    py_object = [1,2,3]
+    cdef PyObject* ptr = <PyObject*>py_object
+    py_object = None   # last reference to list dies here
+
+    # ptr now points to a dead object
+    print(<object>ptr)   # expect a crash here!
+
+Pointers are commonly used when passing objects through C callbacks, e.g.
+
+::
+
+    cdef int call_it_from_c(void* py_function, void* args):
+        py_args = <tuple>args if args is not NULL else ()
+        return (<object>py_function)(*py_args)
+
+    def py_func(a,b,c):
+        print(a,b,c)
+        return -1
+
+    args = [1,2,3]
+
+    call_it_from_c(<PyObject*>py_func, <PyObject*>args)
+
+Once again, care must be taken to keep the objects alive as long as any pointers to them are still in use.
+
+----------
+
+Why does ``**`` on int literals not work (as it seems to do in Pyrex)?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Answer**: It works as expected in recent versions of Cython.
```

Comment:
Seems obsolete - remove?

Reply:
Or link to https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html#power-operator

Reply:
I'd rather remove it all together. There's no need to have FAQs that people won't even risk asking. If it's in the regular documentation, that's where people will rather look instead.